l10n/en/firefox/features/adblocker.ftl

```diff
@@ -0,0 +1,74 @@
+#   $blocking (url) - link to https://support.mozilla.org/kb/content-blocking
+features-adblocker-on-firefox-you-can-use = On { -brand-name-firefox }, you can use <a href="{ $privacy }">Privacy</a> or <a href="{ $blocking }">Content Blocking</a> settings to get even more control over ad trackers that serve you the ads.
+
+features-adblocker-choose-your-level-of-protection = Choose your level of protection
+features-adblocker-to-start-click-on-the = To start, click on the { -brand-name-firefox } menu in the top right-hand corner of your screen. It looks like three lines stacked on top of each other. In the drop-down menu, click on Content Blocking. You should see a blue pop-up with different selections.
+features-adblocker-go-easy-with-standard = Go easy with Standard mode
+
+# Variables:
+#   $url (url) - link to https://www.mozilla.org/firefox/features/private-browsing/
+features-adblocker-if-ads-dont-bother-you = If ads don’t bother you and you don’t mind being followed by trackers and third-party cookies, then the Standard setting should work for you. To get trackers off your tail in Standard mode, use a <a href="{ $url }">Private Browsing</a> window.
+
+features-adblocker-get-tough-with-strict = Get tough with Strict mode
+features-adblocker-if-seeing-too-many-ads = If seeing too many ads ruins your day, then the Strict mode is a better fit. This mode will block known third-party trackers and cookies in all { -brand-name-firefox } windows.
+features-adblocker-do-it-yourself-custom = Do-it-yourself Custom mode
+features-adblocker-the-custom-setting-gives = The Custom setting gives you the ultimate choice. You can decide what you’re blocking, including trackers, cookies and more. If you allow cookies from a website, you’ll automatically be in Custom mode.
+features-adblocker-cover-your-trail-block = Cover your trail, block trackers
+features-adblocker-click-on-the-trackers = Click on the Trackers box and you’ll be able to block trackers in two ways. One way to block trackers is to do it when you’re working in a Private Window. Another way to do it is to block trackers in all windows. Keep in mind that if you choose to always block trackers, some pages might not load correctly.
+features-adblocker-take-a-bite-out-of-cookies = Take a bite out of cookies
+
+# Variables:
+#   $url (url) - link to https://support.mozilla.org/kb/storage
+features-adblocker-cookies-are-sent-by-websites = <a href="{ $url }">Cookies</a> are sent by websites you visit. They live on your computer and monitor what you’ve been doing on a site. When an airline hikes your rates because you’ve looked at plane tickets once that day, that is the handiwork of a cookie.
+
+features-adblocker-in-firefox-you-can-block = In { -brand-name-firefox }, you can block all third-party cookies or just those set by trackers. Be aware that blocking all cookies can break some sites.
+features-adblocker-send-a-do-not-track-signal = { -brand-name-send } a Do Not Track signal
```

Comment:
The `Send` here is a verb. I missed this one. Need a new PR.

Reply:
Opened https://github.com/mozilla-l10n/www-l10n/pull/112 thanks